File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = 9;
        int[] arr = new int[N];
        int max = 0;
        int count = 0;
        int index = 0;
        
        for(int i = 0; i < N; i++) {
            arr[i] = sc.nextInt();
        }
        
        for(int i = 0; i < arr.length; i++) {
            count++;
            
            if(max < arr[i]) {
                max = arr[i];
                index = count;
            }
        }
        System.out.println(max);
        System.out.println(index);
    }
}

File: 백준/Bronze/2562. 최댓값/최댓값.cs
using System;

class Q2562
{
    static void Main(String[] args)
    {
        int N = 9;
        int[] arr = new int[N];
        
        for (int i = 0; i < N; i++)
        {
            arr[i] = int.Parse(Console.ReadLine());
        }
        
        int Max = 0;
        int count = 0;
        int index = 0;
        
        for (int i = 0; i < N; i++)
        {
            count++;
            
            if (Max < arr[i])
            {
                Max = arr[i];
                index = count;
            }
        }
        
        Console.WriteLine(Max);
        Console.WriteLine(index);
    }
}